schemas/produktside/innholdsseksjon/innholdsseksjonReference.ts

```diff
@@ -1,21 +1,21 @@
 /* eslint-disable @typescript-eslint/explicit-module-boundary-types */
 // @ts-nocheck
-import { innholdsseksjon } from "./innholdsseksjon";
 import { InnholdsseksjonPreview, InnholdsseksjonIkon } from "./InnholdsseksjonPreview";
 
 export const innholdsseksjonReferenceName = "innholdsseksjonReference";
+const innholdsseksjonName = "innholdsseksjon";
```

Comment:
sanity likte ikke jeg refererte til innholdsseksjon.name, så måtte lage egen konstant for det her